banzai_nres/dbs.py

```diff
@@ -57,26 +103,41 @@ def populate_phoenix_models(model_location, db_address):
                 banzai.dbs.add_or_update_record(db_session, ResourceFile, {'key': 'phoenix_wavelengths'},
                                                 {'filename': filename, 'location': location, 'key': 'phoenix_wavelengths'})
                 continue
+
+            # Note that there are 25 header keyword value pairs in a standard phoenix model file all in cgs units
+            # FITS headers have 80 character header lines
+            if 's3' in model_location:
+                header_lines = s3.get_object(Bucket=model_bucket, Key=filename,
+                                             Range=f'0-{80 * 25 - 1}')['Body'].read()
+            else:
+                with open(os.path.join(location, filename), 'rb') as f:
+                    header_lines = f.read(80 * 25)
```

Comment:
having the number of lines hardcoded for the header seems fragile, could we perhaps have the number of lines implicitly stored within the model .fits files themselves?